libcxx/test/libcxx/time/time.traits.is.clock/trait.is.clock.compile.verify.cpp

```diff
@@ -0,0 +1,24 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++20
+
+#include <chrono>
```

Comment:
We usually put a synopsis of what is being tested here. Also the convention is the test name to match the section name + class, e.g. 

libcxx/test/libcxx/time/time.traits.is.clock/trait.is.clock.compile.verify.cpp -> libcxx/test/libcxx/time/time.traits/is.clock.compile.verify.cpp

Reply:
Opened #166888 for this.